ansible/files/zeppelin.sh

```diff
@@ -0,0 +1,200 @@
+#!/bin/bash
+
+# Configure Spark
+chmod o+w /mnt /mnt1
+source $HOME/.bashrc
+
+echo "export PYSPARK_PYTHON=$PYSPARK_PYTHON" | sudo tee -a /etc/zeppelin/conf/zeppelin-env.sh
+
+cat << EOF | sudo tee /etc/zeppelin/conf/interpreter.json > /dev/null
```

Comment:
Instead of using a HERE doc, can we move this configuration file to it's own file, sync it to s3, and s3 cp it down during provisioning?